### PR TITLE
{bio}[foss/2020b] phyx v1.3

### PR DESCRIPTION
**Pull request by sdx23**
(created using `eb --new-pr`)
Updated to current version and newer toolchain

### CODE REVIEW

File: easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.01_link_openblas.patch']
+checksums = [
+    'b09677f98f997838b39cd2724463be06d0627ddc90a7e659dc90d4e02db624eb',  # v1.3.tar.gz
+    '8c43454c7ad87cfa9ac1eacef31fb651637f8ae218d6337beabbc98899eb574d',  # phyx-1.01_link_openblas.patch
+]
+
+builddependencies = [
+    ('Autotools', '20200321', '', ('GCCcore', '10.2.0')),
```
Comment (smoors) [inline easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb:23]: adding a subtoolchain is not necessary, EB is smart enough to know this.
```suggestion
    ('Autotools', '20200321'),
```

File: easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'phyx'
+version = '1.3'
+
+homepage = 'https://github.com/FePhyFoFum/%(name)s'
```
Comment (smoors) [inline easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb:7]: please don't use templates in the homepage url, as that makes it difficult to copy-paste into a browser.
```suggestion
homepage = 'https://github.com/FePhyFoFum/phyx'
```

File: easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                                           "pxboot", "pxbp", "pxcat", "pxclsq", "pxcltr", "pxcolt", "pxconsq",
+                                           "pxcontrates", "pxfqfilt", "pxlog", "pxlssq", "pxlstr", "pxmrca",
+                                           "pxmrcacut", "pxmrcaname", "pxnw", "pxrecode", "pxrevcomp",
+                                           "pxrls", "pxrlt", "pxrmk", "pxrms", "pxrmt", "pxrr", "pxs2fa", "pxs2nex",
+                                           "pxs2phy", "pxseqgen", "pxssort", "pxsstat", "pxstrec", "pxsw", "pxt2new",
+                                           "pxt2nex", "pxtcol", "pxtcomb", "pxtlate", "pxtrt", "pxtscale",
+                                           "pxvcf2fa"]],
+    'dirs': [],
+}
+
```
Comment (smoors) [inline easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb:48]: can you add a few `sanity_check_commands` here? 
```
sanity_check_commands = [
    'pxaa2cdn --help',
    ...
]
```

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb (hunk 3 above)
Comment (sdx23) [reply easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb:48]: I'm not a user of that program myself. So, at most I could add stuff like "pfoobar --help". But is that worthwhile one the existence of the binaries is already ensured?

Comment (smoors) [reply easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb:48]: the sanity check commands are meant for a minimal check of the binaries: catching missing linked libraries, etc.
fully testing their correctness is out of scope here.
adding `pfoobar --help` for a couple of binaries is sufficient.

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**